packages/integration-tests/projects/suite-web/tests/metadata/account-metadata.test.ts

```diff
@@ -101,10 +116,13 @@ Hovering over fields that may be labeled shows "add label" button upon which is
             // - why asserting it:  just to make sure that metadata don't send unnecessary amount of request
             cy.waitUntil(() =>
                 cy.task('metadataGetRequests', { provider: f.provider }).then(requests => {
+                    const expectedNumberOfRequests = f.numberOfRequests[startEmuParams.version]
+                        ? f.numberOfRequests[startEmuParams.version]
+                        : f.numberOfRequests.legacy;
```

Comment:
nit, no need to change, just a note – this syntax is useful in those cases instead of ternary with the same condition as the truthy expression:
```suggestion
                    const expectedNumberOfRequests = f.numberOfRequests[startEmuParams.version] ?? f.numberOfRequests.legacy;
```